src/app/templates/AssetInfo.tsx

```diff
@@ -22,7 +22,7 @@ const AssetInfo: FC<AssetInfoProps> = ({ assetSlug }) => {
 
   const metadata = useAssetMetadata(assetSlug);
 
-  if (isTezAsset(asset)) return null;
+  // if (isTezAsset(asset)) return null;
```

Comment:
?

Reply:
tab "Info" for Tez asset is required for new design